src/odemis/gui/cont/tabs/fastem_overview_tab.py

```diff
@@ -65,6 +66,9 @@ def __init__(self, name, button, panel, main_frame, main_data, vp, view_controll
         self.tab_data.semStream = sem_stream
 
         # Buttons of the calibration panel
+        panel.lbl_optical_autofocus.SetForegroundColour(FG_COLOUR_MAIN)
+        panel.lbl_sem_autofocus.SetForegroundColour(FG_COLOUR_MAIN)
+        panel.lbl_autobrightness_contrast.SetForegroundColour(FG_COLOUR_MAIN)
```

Comment:
Looking at the XRC file, it seems that quite a few widgets just have a `<fg>#DDDDDD</fg>` (which is the FG_COLOUR_MAIN). I think we should do the same for these 3 StaticText, instead of changing their colour in the code.